solutions/src/test/java/ru/sberbank/school/task10/ScalableThreadPoolTest.java

```diff
@@ -0,0 +1,87 @@
+package ru.sberbank.school.task10;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.jupiter.api.Assertions;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class ScalableThreadPoolTest {
+    private ThreadPool threadPool;
+
+    @Before
+    public void init() {
+        threadPool = new ScalableThreadPool(3, 12);
+        threadPool.start();
+    }
+
+    @After
+    public void stop() {
+        threadPool.stopNow();
+    }
+
+    @Test(timeout = 500)
+    public void NormalWork() throws InterruptedException {
```

Comment:
Method name &#39;NormalWork&#39; must match pattern &#39;^[a-z][a-z0-9][a-zA-Z0-9_]*$&#39;.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1564816738>*